graph/client/src/app/feature-projects/project-list.tsx

```diff
@@ -49,7 +49,7 @@ function groupProjectsByDirectory(
         ? workspaceLayout.appsDir
         : workspaceLayout.libsDir;
     const directories = parseParentDirectoriesFromFilePath(
-      project.data.root,
+      (project.data as any).root,
```

Comment:
Why is this cast necessary now? `project.data` should always have a root prop if its an Nx project. Is there a check of some kind missing to ensure its a ProjectGraphProjectNode instead of an external node?